.storybook/preview.js

```diff
@@ -41,7 +41,11 @@ addParameters({
     options: {
         storySort: (a, b) =>
             a[1].kind === b[1].kind
-            ? 0
-            : a[1].id.localeCompare(b[1].id, undefined, { numeric: true }),
+                ? 0
+                : a[1].id.localeCompare(b[1].id, undefined, { numeric: true }),
+    },
+    jsx: {
+        filterProps: val => val !== undefined,
+        showDefaultProps: false,
```

Comment:
These changes are not directly related to the table, but they do clean up the JSX storybook tab quite nicely:
- Omit props if their value equals `undefined`
- Omit default props

This was also [discussed in Slack](https://dhis2.slack.com/archives/CBM8LNEQM/p1612943303197900) and approved there.

If others prefer to introduce these changes in a separate PR, I would be OK with that.